public/assets/app.css

```diff
@@ -363,7 +363,7 @@ table a {
 }
 
 table tr td:first-child {
-  width: 100px;
+  width: 150px;
```

Comment:
why the 150px? Wouldn’t it make sense to give it the same width as the Amount input?